src/Microsoft.Dnx.Runtime/FrameworkReferenceResolver.cs

```diff
@@ -232,12 +238,105 @@ private static FrameworkInformation GetFrameworkInformation(FrameworkName target
             return GetFrameworkInformation(version, targetFramework);
         }
 
+        private static FrameworkInformation GetLegacyFrameworkInformation(FrameworkName targetFramework, string referenceAssembliesPath)
+        {
+            var frameworkInfo = new FrameworkInformation();
+            if (!RuntimeEnvironmentHelper.IsMono)
+            {
+                // Always grab .NET 2.0 data
+                var searchPaths = new List<string>();
+                var net20Dir = Path.Combine(Environment.GetEnvironmentVariable("WINDIR"), "Microsoft.NET", "Framework", "v2.0.50727");
+
+                if (!Directory.Exists(net20Dir))
+                {
+                    return null;
```

Comment:
Pretty much nothing :). A distinction is kinda needed because there's a bunch of pre-computed framework data but we still need to know if the framework is actually installed (via a Directory.Exists call).